### DISCUSSION TITLE
Detected: HackTool:Win64/Patcher!MSR

### DISCUSSION THREAD

**Discussion by jessie-apollotech** [valinet/ExplorerPatcher#2937] [Q&A]
detected as malware by microsoft

file: C:\Program Files\ExplorerPatcher\ep_setup.exe

regkey: HKLM\SOFTWARE\MICROSOFT\WINDOWS\CURRENTVERSION\UNINSTALL\{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ExplorerPatcher

uninstall: HKLM\SOFTWARE\MICROSOFT\WINDOWS\CURRENTVERSION\UNINSTALL\{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ExplorerPatcher

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2873

This is a false positive.

Update your windows defender signatures by going to settings -> updates & security -> windows security
Then click on virus & threat protection
Under virus & threat protection updates, click on check for updates
Then click on the button named check for updates

Then if it is still being detected as malicious, submit it as a false positive to Microsoft: https://www.microsoft.com/en-us/wdsi/filesubmission

**Comment by Amrsatrio** [comment]
Submitting a false positive report won't do anything. They will still flag this as a virus. They don't care if it's open source or not, seems like they're making decisions based on the behavior. Anyways, please download 63.4 and not 63.2 or 63.3. 63.4 is currently not yet flagged by MS.

**Reply by bickle** [reply]
I'm only seeing 63.3 on https://github.com/valinet/ExplorerPatcher/releases

Anyway, is the behavior that is getting flagged something that we could live without?

**Reply by pyrates999** [reply]
63.3 was a pre-release and superseded with the release of 64.1.

**Reply by bickle** [reply]
Sorry, long day. Got my numbers mixed up (63.3/64.3). 64.3 is also getting flagged despite being prerelease, so my question regarding behavior still stands. I'd rather have a scaled back version than a version I can't install

**Comment by Javican** [comment]
Agree , i have submited multiple reports in the past using my enterprise contract and in this case they came back that its still malware.
The new version is flagged as potential unwanted behavior!

**Reply by sheederj5** [reply]
Did you get the update to work? I've updated defender and windows but still getting flagged.

**Comment by Javican** [comment]
Every single version gets flagged, i even got a Windows cloud defender " Non Enterprise software" or something like that alert
